Source/Scene/QuadtreePrimitive.js

```diff
@@ -247,6 +283,7 @@ define([
         }
 
         var i;
+        var j;
```

Comment:
At some point in this function, the quadtree should know (or could know) if we reached a leaf node, and we could use this to let the tile's billboards know that we reached the maximum level and they don't need updates.